### ISSUE TITLE
Amount permanently stuck as unavailable after activation attempt

### ISSUE THREAD

**Issue by lorexodus** [massnetorg/MassNet-wallet#14]
I tried to activate a number of Chia plots and the transactions were made, but they never appeared on the blockchain (48 hours later).
A large amount of the total balance (a lot more then the activation cost) is now permanently shown as unavailable in the wallet.
After a full resync the amount appeared to be available again, but after a transaction attempt, the complete balance became permanently unavailable and the transaction also didn't appear on the blockchain. After import to another wallet the earlier large amount that became unavailable after the activation attempt was again unavailable permanently. Is there a way to repair this?

**Comment by silentsc** [comment]
Yup same here, unfortunately once after the passphase, it just went on and on without indicating that the activation were successful, and the amount were deducted from the balance. No action done and no activation on the files. I'm kind of lost and I've used the online web wallet to re-check the files and it states invalidated (no activation done) ... so now where's the mass balance went to?
